.github/workflows/lint.yml

```diff
@@ -25,6 +25,8 @@ jobs:
       - name: revive lint
         uses: morphy2k/revive-action@v2
         with:
+          config: .revive.toml
+          exclude: "*.pb.go;*.pb.gw.go;*.pulsar.go;swagger/..."
           path: "./..."
```

Comment:
This step generates annotations on files in pull requests, and mirrors the local output of `make lint`.

Neither revive `warnings` nor `errors` will appear as a CI fail though, so any rules we want to require should appear in the `golangci.yml` config.